spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe  UsersController, type: :controller do
+  describe "GET new" do
+    it "assigns a blank article to the view" do
+      get :new
+      expect(assigns(:user)).to be_a_new(User)
+    end
+  end
+
+  describe "POST #create" do
+   context "with valid attributes" do
+     it "create new contact" do
+      post :create, params: {user: FactoryBot.attributes_for(:user)}
+      expect(response).to redirect_to root_url
+       expect(User.count).to eq(1)
```

Comment:
Tách 2 `it` ra em nhé
ko nên gộp thế này
`post :create, params: {user: FactoryBot.attributes_for(:user)}` để trong before sau `context` line 12